website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -80,6 +80,51 @@ Use this variant only when you need to put interactive elements such as a button
 
 ![Example of toggle content](/assets/components/accordion/toggle-content-interactive.png =800x*)
 
+## Sizes
+
+Our Accordion comes in three sizes: `small`, `medium`, and `large`. We recommend using `medium`, but use whichever size best fits your UI’s needs.
+
+![Image here of different sizes - only card variant](/assets/components/accordion/accordion-size-range.png)
+
+!!! Dont
+
+The size is set at the Accordion (group) level, so AccordionItem sizes are not interchangeable.
```

Comment:
[suggestion] Maybe use language that specifically highlights this as a "dont" because it's _technically_ possible in the Figma component to use different sizes.

This isn't 100% there, but a bit closer.

```suggestion
The size is set at the Accordion (group) level, don't mix different sizes of the AccordionItem.
```